app/jobs/appointment_notification/schedule_experiment_reminders.rb

```diff
@@ -10,11 +10,18 @@ def perform
 
     notifications = Notification.due_today
     next_messaging_time = Communication.next_messaging_time
-
     logger.info "scheduling #{notifications.count} notifications that are due with next_messaging_time=#{next_messaging_time}"
     notifications.each do |notification|
       notification.status_scheduled!
       AppointmentNotification::Worker.perform_at(next_messaging_time, notification.id)
+    rescue => e
+      Sentry.capture_message("Scheduling notification failed",
+        extra: {
+          reminder: notification,
```

Comment:
Let's send just the notification ID? Not sure how the notification will get serialised.

Reply:
I like this idea.